en/components/grids_templates/excel_style_filtering.md

```diff
@@ -337,6 +337,347 @@ If you want to keep the sorting, moving, pinning and hiding features of the colu
 
 <div class="divider--half"></div>
 
+###Styling
+
+To get started with styling the filtering row, we need to import the `index` file, where all the theme functions and component mixins live:
+
+```scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+``` 
+
+The excel style filtering dialogs takes its background color from the grid's theme, using the `filtering-row-background` parameter. So in order to change the background we need to create a custom theme:
```

Comment:
1. The excel style filtering dialog~~s~~ ...
2. ~~So~~ **I**n order to change ...

Reply:
Addressed.